src/main/java/games/strategy/triplea/attachments/TriggerAttachment.java

```diff
@@ -2533,75 +2534,75 @@ public static void triggerVictory(final Set<TriggerAttachment> satisfiedTriggers
     }
   }
 
-  public static Match<TriggerAttachment> prodMatch() {
+  public static Predicate<TriggerAttachment> prodMatch() {
```

Comment:
Just wondering if you plan on eventually renaming methods/variables like this to remove/replace the word "match"?

Reply:
Didn't plan to replace it as part of this series, but I'd say this is easily done if wanted.